lib/service/proxy_settings.go

```diff
@@ -113,4 +120,30 @@ func (p *proxySettings) setProxyPublicAddressesSettings(settings *webclient.Prox
 	if len(p.cfg.Proxy.MySQLPublicAddrs) > 0 {
 		settings.DB.MySQLPublicAddr = p.cfg.Proxy.MySQLPublicAddrs[0].String()
 	}
+	settings.DB.PostgresPublicAddr = p.getPostgresPublicAddr()
```

Comment:
Since you added this, is the block above on lines 117-119 still needed? Seems like it can be removed now:

```go
	if len(p.cfg.Proxy.PostgresPublicAddrs) > 0 {
		settings.DB.PostgresPublicAddr = p.cfg.Proxy.PostgresPublicAddrs[0].String()
	}
```